### PR TITLE
Issue 63 stone sorcery

### PR DESCRIPTION
**Pull request by professionalzack**
Okay so I added the content for the stone sorcerer to the bet of my abilities. let me know what you think. this is for issue #63

### CODE REVIEW

File: unearthed-arcana/2017/20170206.xml [hunk 1]
```diff
@@ -223,5 +223,179 @@
...
+          <stat name="HP" value="1" level="15" />
+          <stat name="HP" value="1" level="16" />
+          <stat name="HP" value="1" level="17" />
+          <stat name="HP" value="1" level="18" />
+          <stat name="HP" value="1" level="19" />
+          <stat name="HP" value="1" level="20" />
+        </rules>
+      </element>
+
+      <element name="Stone Aegis" type="Archetype Feature" source="Unearthed Arcana" id="ID_WOTC_UA_ARCHETYPE_FEATURE_SEA_SORCERER_SHIFTING_FORM">
```
Comment (swdriessen) [inline unearthed-arcana/2017/20170206.xml]: This element has the ID of a copy/pasted element.

File: unearthed-arcana/2017/20170206.xml [hunk 2]
```diff
@@ -223,5 +223,179 @@
...
+      </element>
+
+      <element name="Stone's Edge" type="Archetype Feature" source="Unearthed Arcana" id="ID_WOTC_UA_ARCHETYPE_FEATURE_STONE_SORCERER_STONES_EDGE">
+        <supports />
+        <description>
+          <p>Starting	at	14th	level,	your	mastery	of	earth	magic	allows	you	to	add	the	force	of	elemental	earth	to	your	spells.	When	you	cast	a	spell	that	deals	damage,	choose	one	creature	damaged	by that	spell	on	the	round	you	cast	it.	That	creature	takes	extra	force	damage	equal	to	half	your	sorcerer	level.	This	feature	can	be	used	only	once	per	casting	of	a	spell.</p>
+        </description>
+        <sheet>
+          <description>
+            When	you	cast	a	spell	that	deals	damage,	choose	one	creature	damaged	by that	spell	on	the	round	you	cast	it.	That	creature	takes	extra	%hacky half level% force	damage.
```
Comment (swdriessen) [inline unearthed-arcana/2017/20170206.xml]: You can use the **level:sorcerer:half** here, this hacky half level was used when these level statistics were not yet generated by the builder.

File: unearthed-arcana/2017/20170206.xml (hunk 1 above)
Comment (professionalzack) [reply unearthed-arcana/2017/20170206.xml]: got it

File: unearthed-arcana/2017/20170206.xml (hunk 2 above)
Comment (professionalzack) [reply unearthed-arcana/2017/20170206.xml]: oh great! thanks

**Comment by professionalzack** [comment]
Okay pushed an updated commit called "requested changes" that solves all the stated issues. thanks !

**Comment by swdriessen** [comment]
Thanks, there are some tabs still in there. I'll see if i can change them and i'll merge it. 

I assume they came from copy/pasting description. Best to full find/replace in a seperate window before pasting in the description.

**Comment by swdriessen** [comment]
Thanks!

**Pull request merged**